### PR TITLE
Config: Parsing configuration

### PR DESCRIPTION
**Pull request by tomMoulard**

### CODE REVIEW

File: rules-fail2ban.yaml [hunk 1]
```diff
@@ -34,3 +34,28 @@ http:
...
+                chain: ""          
+                port:
+                  - 0
+                  - 10000             
+                fail2banAgent: ""    
+                banaction: ""
+                banactionAllports: ""
+                actionAbuseipdb: ""  
+                action: ""  
+              }
```
Comment (tomMoulard) [inline rules-fail2ban.yaml:61]: ?

File: rules-fail2ban.yaml [hunk 2]
```diff
@@ -34,3 +34,28 @@ http:
             IP:
                 - "::1"
                 - "127.0.0.1"
+            Rules:
```
Comment (tomMoulard) [inline rules-fail2ban.yaml:37]: Wouldn't be better that `Rules` be on the same level as `Whitelist` or `Blacklist` in `Dev` ?

File: rules-fail2ban.yaml (hunk 1 above)
Comment (MikeSPROW) [reply rules-fail2ban.yaml:61]: I have juste change the name to respect the Golang syntax

File: rules-fail2ban.yaml (hunk 2 above)
Comment (MikeSPROW) [reply rules-fail2ban.yaml:37]: Yes i will change that

**Pull request merged**